### PR TITLE
Disable usage statistics when running ATH tests

### PR DESCRIPTION
**Pull request by i386**
# Description
disable stats when running ATH

### CODE REVIEW

**Comment by i386** [comment]
@imeredith is this right?

File: acceptance-tests/run.sh [hunk 1]
```diff
@@ -37,8 +37,7 @@ if [ "${RUN_SELENIUM}" == "true" ]; then
     ./runner/scripts/start-selenium.sh
 fi
 
-
-EXECUTION="env JENKINS_JAVA_OPTS=\"${JENKINS_JAVA_OPTS}\" ${ATH_SERVER_HOST} ${ATH_SERVER_PORT} BROWSER=phantomjs LOCAL_SNAPSHOTS=${LOCAL_SNAPSHOTS} ${PLUGINS} PLUGINS_DIR=../runtime-plugins/runtime-deps/target/plugins-combined PATH=./node:./node/npm/bin:./node_modules/.bin:${PATH} mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B -Dmaven.test.failure.ignore ${MAVEN_SETTINGS} test ${PROFILES} ${TEST_TO_RUN}"
+EXECUTION="env JENKINS_JAVA_OPTS=\"${JENKINS_JAVA_OPTS}\" ${ATH_SERVER_HOST} ${ATH_SERVER_PORT} BROWSER=phantomjs LOCAL_SNAPSHOTS=${LOCAL_SNAPSHOTS} ${PLUGINS} PLUGINS_DIR=../runtime-plugins/runtime-deps/target/plugins-combined PATH=./node:./node/npm/bin:./node_modules/.bin:${PATH} mvn -Dhudson.model.UsageStatistics.disabled=true -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B -Dmaven.test.failure.ignore ${MAVEN_SETTINGS} test ${PROFILES} ${TEST_TO_RUN}"
```
Comment (i386) [inline acceptance-tests/run.sh]: added `-Dhudson.model.UsageStatistics.disabled=true `

**Comment by michaelneale** [comment]
edited description - please do tweak it when opening a PR so we can see it in change log in future.

**Comment by i386** [comment]
@michaelneale np

**Comment by imeredith** [comment]
🐝  lgtm

**Comment by i386** [comment]
@imeredith so this should pass the property through to Jenkins?

**Pull request merged**